packages/maskbook/src/plugins/Snapshot/hooks/useProposal.ts

```diff
@@ -3,8 +3,8 @@ import type { Proposal, ProposalMessage } from '../types'
 import { useSuspense } from '../../../utils/hooks/useSuspense'
 
 const cache = new Map<string, [0, Promise<void>] | [1, { proposal: Proposal; message: ProposalMessage }] | [2, Error]>()
-export function proposalErrorRetry() {
-    cache.forEach(([status], id) => status === 2 && cache.delete(id))
+export function proposalRetry() {
+    Array.from(cache.keys()).forEach((id) => cache.delete(id))
```

Comment:
```
Array.from(cache.keys()).forEach(cache.delete)
```
This simple writing will cause error: `Uncaught (in promise) TypeError: Method Map.prototype.delete called on incompatible receiver undefined`